cli/test/org/partiql/cli/functions/ReadFileTest.kt

```diff
@@ -77,7 +77,55 @@ class ReadFileTest {
         val actual = function.call(env, args).ionValue
         val expected = "[{_1:\"1\",_2:\"2\"}]"
 
-        assertEquals(actual, ion.singleValue(expected))
+        assertEquals(ion.singleValue(expected), actual)
+    }
+
+    @Test
```

Comment:
Should `tsv` files also support the use of a header row?  If so, I don't see any tests for `tsv` files that include a header row.

Reply:
Will add test for reading tsv files with a header in the next revision.